services/geocoding.py

```diff
@@ -27,15 +24,21 @@ def enhance(self, entries: List[PointOfInterest]) -> List[PointOfInterest]:
         For entries with missing lat/lng but present address
         we can find lat/lng using a geocoding API.
         """
+        #Create client instance of gmaps
+        gmaps = googlemaps.Client(key=self.settings.developer_key)
+
         addresses_to_geocode: List[str] = [
             poi.address
             for poi in entries
             if not poi.lat or not poi.lng
         ]
 
         coordinates: Mapping[str, Point] = {}
-        # TODO: use a geocoding API to fill the coordinates for each address
 
+        for address in addresses_to_geocode:
+            latLng = gmaps.geocode(address)
```

Comment:
Let's extract `gmaps.geocode(address)` into `make_geocoding_request` (see `google_sheets.py`) so that we can fake it in unit tests